src/Stringy.php

```diff
@@ -463,8 +463,21 @@ public function hasUpperCase()
      */
     public function htmlDecode($flags = ENT_COMPAT)
     {
-        $str = html_entity_decode($this->str, $flags, $this->encoding);
-
+        if (function_exists('html_entity_decode')) {
```

Comment:
I still don't understand, when and why would `html_entity_decode` not be used? It's part of the standard library.